rskj-core/src/integrationTest/java/co/rsk/snapshotsync/SnapshotSyncIntegrationTest.java

```diff
@@ -89,33 +90,37 @@ public void whenStartTheServerAndClientNodes_thenTheClientWillSynchWithServer()
         serverNode.startNode();
         ThreadTimerHelper.waitForSeconds(30);
 
+        JsonNode serverBestBlockResponse = OkHttpClientTestFixture.getJsonResponseForGetBestBlockMessage(portServerRpc, "latest");
+        String serverBestBlockNumber = serverBestBlockResponse.get(0).get("result").get("number").asText();
```

Comment:
In theory, any of these `.get(0).get("result").get("number")` could fail, right?